crates/templates/src/res/pages/login.html

```diff
@@ -33,31 +33,33 @@
   <section class="flex items-center justify-center flex-1">
     <form method="POST" class="grid grid-cols-1 gap-6 w-96 m-2">
       <div class="text-center">
-        <h1 class="text-lg text-center font-medium">Log in</h1>
-        {% if next and next.kind == "continue_authorization_grant" %}
-          <p>to continue to <em>{{ next.grant.client.client_name | default(value=next.grand.client.client_id) }}</em></p>
-        {% else %}
-          <p>Use your existing account</p>
-        {% endif %}
+        <h1 class="text-lg text-center font-medium">Sign in</h1>
+        <p>Please sign in to continue:</p>
       </div>
       <input type="hidden" name="csrf" value="{{ csrf_token }}" />
       {# TODO: errors #}
       {{ field::input(label="Username", name="username") }}
       {{ field::input(label="Password", name="password", type="password") }}
-      {{ button::button(text="Next") }}
-      {{ button::link_text(text="Create account", href=register_link) }}
+      {% if next and next.kind == "continue_authorization_grant" %}
+        <div class="grid grid-cols-2 gap-4">
+          {{ back_to_client::link(
+            text="Cancel",
+            class=button::text_class(),
```

Comment:
`class=button::plain_error_class()` would probably make it prettier

Reply:
I think I changed it from the solid red/alert block to the text as I felt it was distracting from the Next action.

How about this instead?

<img width="445" alt="image" src="https://user-images.githubusercontent.com/6955675/167427277-a2f45f7b-6613-49db-b021-be624cbf85df.png">